packages/mint-components/src/components/sqm-logout-current-user/sqm-logout-current-user.feature

```diff
@@ -0,0 +1,19 @@
+@author:noah
+@owner:noah
+
+Feature: Logout Current User
+
+  Background:
+    Given a user viewing an instant access widget
+
+  Scenario: A user on an instant access widget
+    Given the user is identified
+    And the user is logged in
+    Then the component identifies the user by email
```

Comment:
And what does it display?

Reply:
Specified "by displaying their email in an ICU string"